2024-06-29 10:23:07,289 INFO    MainThread:46794 [wandb_setup.py:_flush():76] Current SDK version is 0.17.3
2024-06-29 10:23:07,289 INFO    MainThread:46794 [wandb_setup.py:_flush():76] Configure stats pid to 46794
2024-06-29 10:23:07,290 INFO    MainThread:46794 [wandb_setup.py:_flush():76] Loading settings from /home/student/.config/wandb/settings
2024-06-29 10:23:07,290 INFO    MainThread:46794 [wandb_setup.py:_flush():76] Loading settings from /home/student/R/fl/PFLlib/system/wandb/settings
2024-06-29 10:23:07,290 INFO    MainThread:46794 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-06-29 10:23:07,290 INFO    MainThread:46794 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-06-29 10:23:07,290 INFO    MainThread:46794 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'system/main.py', 'program_abspath': '/home/student/R/fl/PFLlib/system/main.py', 'program': '/home/student/R/fl/PFLlib/system/main.py'}
2024-06-29 10:23:07,290 INFO    MainThread:46794 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-06-29 10:23:07,290 INFO    MainThread:46794 [wandb_init.py:_log_setup():520] Logging user logs to /home/student/R/fl/PFLlib/system/wandb/run-20240629_102307-gtkzveyr/logs/debug.log
2024-06-29 10:23:07,290 INFO    MainThread:46794 [wandb_init.py:_log_setup():521] Logging internal logs to /home/student/R/fl/PFLlib/system/wandb/run-20240629_102307-gtkzveyr/logs/debug-internal.log
2024-06-29 10:23:07,291 INFO    MainThread:46794 [wandb_init.py:init():560] calling init triggers
2024-06-29 10:23:07,291 INFO    MainThread:46794 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-06-29 10:23:07,291 INFO    MainThread:46794 [wandb_init.py:init():610] starting backend
2024-06-29 10:23:07,291 INFO    MainThread:46794 [wandb_init.py:init():614] setting up manager
2024-06-29 10:23:07,298 INFO    MainThread:46794 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-06-29 10:23:07,299 INFO    MainThread:46794 [wandb_init.py:init():622] backend started and connected
2024-06-29 10:23:07,307 INFO    MainThread:46794 [wandb_init.py:init():711] updated telemetry
2024-06-29 10:23:07,322 INFO    MainThread:46794 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-06-29 10:23:07,978 ERROR   MainThread:46794 [wandb_init.py:init():770] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2024-06-29 10:23:07,978 ERROR   MainThread:46794 [wandb_init.py:init():1181] error in wandb.init()
Traceback (most recent call last):
  File "/home/student/.local/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1171, in init
    return wi.init()
  File "/home/student/.local/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 776, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2024-06-29 10:23:09,181 WARNING MsgRouterThr:46794 [router.py:message_loop():77] message_loop has been closed
